src/components/MapStyleInputWrapper.svelte

```diff
@@ -156,12 +188,25 @@
     }));
 
     setSelectedValue();
+    setRendererOptions();
+    setRendererValue();
```

Comment:
Could we just combine `setRendererValue` and `setRenderOptions` since they're always paired anyway?